src/javascript/binary/pages/trade/socket.js

```diff
@@ -40,17 +40,26 @@ var TradeSocket = (function () {
         }
```

Comment:
tradesocket is changed, so you probably need to adapt 

Related PR:
https://github.com/binary-com/binary-static/pull/707